src/support/logger/logger.c

```diff
@@ -145,6 +143,22 @@ void logger_log(Logger* logger, LogLevel level, const gchar* fileName,
     }
 }
 
+void logger_setLevel(Logger* logger, LogLevel level) {
+    if (!logger) {
+
```

Comment:
Was this empty line intentional? If so, and you want to keep the if statement, at least add a comment.

Reply:
Added a comment